control.lua

```diff
@@ -52,6 +52,8 @@ script.on_event(defines.events.on_gui_click, TLBE.GUI.onClick)
 script.on_event(defines.events.on_gui_selection_state_changed, TLBE.GUI.onSelected)
 script.on_event(defines.events.on_gui_text_changed, TLBE.GUI.onTextChanged)
 script.on_event(defines.events.on_gui_checked_state_changed, TLBE.GUI.onStateChanged)
+script.on_event(defines.events.on_gui_confirmed, TLBE.GUI.onGuiConfirmed)
+script.on_event(defines.events.on_gui_leave, TLBE.GUI.onGuiConfirmed)
```

Comment:
on_gui_leave is when the mouse cursor leaves the element. It only works if you keep the mouse cursor in the element while typing... I doubt that this is going to help with the UX a lot...

Reply:
No. I was trying it out of desperation and hope. It doesn't actually help.

Reply:
Done